src/GameState/convertBoardToBoardCoordinate.tsx

```diff
@@ -0,0 +1,16 @@
+export type BoardCoordinate = { x: number; y: number; value: number };
+
+export const convertBoardToBoardCoordinate = (board: [][]) => {
+  let result: BoardCoordinate[] = [];
+
+  board.map((row: [], index: number) => {
```

Comment:
Why do you use a map if you don't return anything?

Reply:
I think it's better to use two for loops. Or array.reduce combined with a forEach.